pkg/apis/triggers/v1alpha1/event_listener_validation.go

```diff
@@ -19,11 +19,28 @@ package v1alpha1
 import (
 	"context"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/dynamic"
 	"knative.dev/pkg/apis"
 )
 
 // Validate EventListener.
 func (t *EventListener) Validate(ctx context.Context) *apis.FieldError {
+	clientset := ctx.Value("clientSet").(dynamic.Interface)
```

Comment:
I thought I left a comment previously?? Anyway, can we get some unit tests for this?

Reply:
Done